aws/provider.go

```diff
@@ -20,6 +20,9 @@ import (
 	"github.com/pkg/errors"
 )
 
+// version of the Terraform provider, this is automatically changed with the 'make update-terraform-provider'
+const version = "4.9.0"
```

Comment:
```suggestion
const version = "5.1.0"
```

lets use the latest one ;)